8Knot/queries/user_groups_query.py

```diff
@@ -0,0 +1,146 @@
+import logging
+from app import celery_app, augur
+import pandas as pd
+from cache_manager.cache_manager import CacheManager as cm
+import io
+import datetime as dt
+from sqlalchemy.exc import SQLAlchemyError
+import redis
+import json
+
+# DEBUGGING
+import os
+
+QUERY_NAME = "USER_GROUPS_QUERY"
+
+
+@celery_app.task(
+    bind=True,
+)
+def user_groups_query(self, user_id):
```

Comment:
Add comments about why this query is different that the others

Reply:
++